lcd.cpp

```diff
@@ -430,12 +434,16 @@ void LCD_drive_menu() {
 void LCD_driveX() {
   if (lcd_click_now) MENU_POP();
 
+  // if knob was spun quickly, then increment quicker
+  int multiplier = 1;
+  if ( lcd_turns_to_process > 2 ) multiplier = 10;
+  
   float offset[3];
-  offset[0] = axies[0].pos + lcd_turn > 0 ? 1 : -1;
+  offset[0] = axies[0].pos + (lcd_turns_to_process * multiplier);
```

Comment:
lcd_turns_to_processs could be 10.  the if() and the multiplier wouldn't be needed every loop.

Reply:
lcd_turns_to_process depends on the number of turns made on the knob before the LCD_update routine runs.  There could be multiple interrupts because someone it turning the knob quickly, and the interrupt routine just counts the turns.  When LCD_update runs again (asynchronously to counting the turns), then LCD_update will store the current number of turns (lcd_turns) that occurred since the last time LCD_update ran in lcd_turns_to_process.  It doesn't just use the lcd_turn count since that could change during LCD_update processing if an interrupt occurs.  If the person is turning the knob quickly, then I made this code so that it would change the number quicker (10x).  They can do fine tuning by turning the knob slowly, or quick large changes (10x) by turning quickly.  This should just be intuitive for a user - they need a big change and turn quickly and the value just increments at 10x rate without them having to know  its happening.